peer.go

```diff
@@ -1693,6 +1693,11 @@ out:
 				continue
 			}
 
+			// Inform the ChannelNotifier that this channel has transitioned
+			// from pending open to open.
+			p.server.channelNotifier.NotifyOpenChannelEvent(newChan.IdentityPub,
```

Comment:
Would a better place for this be in the `fundingmanager`? If we want this to notify about the channel being "marked open" in the database, then it could be done here: https://github.com/lightningnetwork/lnd/blob/441307e8f7ede6f15ea7c1897de732ade6544ab9/fundingmanager.go#L1935

Reply:
Agreed would be nice to move to the `fundingManager`. Would also get rid of another instance of usage of the server's "God" pointer.

Reply:
fixed :)